walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,85 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
 
 public final class FindMeetingQuery {
+  private static int DAY_LENGTH = 24*60;
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    List<TimeRange> possibleTimes = new ArrayList<TimeRange>();
+    Set<String> attendees =  new HashSet<String>();
+    int meetingDur = (int)request.getDuration();
+    attendees.addAll(request.getAttendees());
+    if (meetingDur > DAY_LENGTH){
+      return possibleTimes;
+    }
+    if (events.isEmpty() || attendees.isEmpty()){
+      possibleTimes.add(TimeRange.WHOLE_DAY);
+      return possibleTimes;
+    }
+    /*A time is considered to be busy when an attendee is in an event & the meeting request*/
```

Comment:
For inline comments, use `\\` instead of javaDoc style `/* ... */`

Reply:
Removed these in latest PR (replaced w helper function)